client/app/css/main.css

```diff
@@ -65,6 +65,102 @@ a:hover,
   opacity: 1;
 }
 
+.uib-datepicker-popup {
+  padding: 5;
+  margin: 0;
+}
+
+.uib-datepicker {
+  padding: 5px;
+}
+
+.uib-button-bar > .btn-group > .btn-info {
+  display: none;
+}
+
+.uib-button-bar > .btn-success {
+  display: none;
+}
+
+[dir] .uib-datepicker th,
+[dir] .uib-datepicker td {
+  white-space: nowrap;
+  min-width: 0;
+  padding: 2px;
+}
+
+[dir] .dropdown-multiselect button::after {
+  border: 0;
+  content: "";
+  vertical-align: 0;
+}
+
+.dropdown-multiselect button {
+  padding: 0;
+  width: 18px;
+  height: 20px;
+  background-image: url("data:image/svg+xml,<?xml version='1.0' encoding='UTF-8'?><svg xmlns='http://www.w3.org/2000/svg' xml:space='preserve' viewBox='0 0 971.99 971.99'><path d='M370.22 459.3c10.2 11.1 15.8 25.6 15.8 40.6v442c0 26.601 32.1 40.101 51.1 21.4L560.42 822c16.5-19.8 25.6-29.601 25.6-49.2V500c0-15 5.7-29.5 15.8-40.601l353.8-383.9c26.5-28.8 6.101-75.5-33.1-75.5h-873c-39.2 0-59.7 46.6-33.1 75.5l353.8 383.8z'/></svg>");
+  background-repeat: no-repeat;
+}
+
+[dir] .uib-datepicker button {
+  background: #fff;
+  color: grey;
+}
+
+[dir] .full button {
+  background: #e6e6e6;
+  color: #2866a2;
```

Comment:
this and the other colors used do not match the brand guidelines of the application and the other colors already used in the bootstrap framework:

https://github.com/globaleaks/GlobaLeaks/blob/main/brand/globaleaks-brand-guidelines.pdf